ci/ray_ci/utils.py

```diff
@@ -25,6 +28,35 @@ def shard_tests(
     return bazel_sharding.main(test_targets, index=shard_id, count=shard_count)
 
 
+def docker_login(docker_ecr: str) -> None:
```

Comment:
I feel that I have reviewed this some time before?

Reply:
lol yes, I removed it at some point but now need it again

Reply:
could you add a unit test for this function?